Ntuplizer/submit_all_opt.py

```diff
@@ -92,16 +92,18 @@ def main():
     print "1)options.luminosity ", options.luminosity
 
     if options.luminosity :
-        config.Data.splitting = 'Automatic' #'EventAwareLumiBased' #LumiBased'
+        #config.Data.splitting = 'Automatic' #'EventAwareLumiBased' #LumiBased'
+        config.Data.splitting = 'FileBased'
```

Comment:
Data should be submitted with the EventAwareLumiBased or Automatic. Also to remove duplicates.